Deployment/Jenkins/master/Master-Ec2-Instance.groovy

```diff
@@ -0,0 +1,281 @@
+                  },
+                  {
+                      "ParameterKey": "JenkinsOsPrepScriptUrl",
+                      "ParameterValue": "${env.JenkinsOsPrepScriptUrl}"
+                  },
+                  {
+                      "ParameterKey": "JenkinsRepoKeyURL",
+                      "ParameterValue": "${env.JenkinsRepoKeyURL}"
+                  },
+                  {
+                      "ParameterKey": "JenkinsRepoURL",
+                      "ParameterValue": "${env.JenkinsRepoURL}"
+                  },
+                  {
+                      "ParameterKey": "JenkinsRpmName",
+                      "ParameterValue": "${env.JenkinsRpmName}"
+                  },
+                  {
+                      "ParameterKey": "KeyPairName",
+                      "ParameterValue": "${env.KeyPairName}"
+                  },
+                  {
+                      "ParameterKey": "NoPublicIp",
+                      "ParameterValue": "${env.NoPublicIp}"
+                  },
+                  {
+                      "ParameterKey": "NoReboot",
+                      "ParameterValue": "${env.NoReboot}"
+                  },
+                  {
+                      "ParameterKey": "NoUpdates",
+                      "ParameterValue": "${env.NoUpdates}"
+                  },
+                  {
+                      "ParameterKey": "PipIndexFips",
+                      "ParameterValue": "${env.PipIndexFips}"
+                  },
+                  {
+                      "ParameterKey": "PipRpm",
+                      "ParameterValue": "${env.PipRpm}"
+                  },
+                  {
+                      "ParameterKey": "ProvisionUser",
+                      "ParameterValue": "${env.ProvisionUser}"
+                  },
+                  {
+                      "ParameterKey": "PyStache",
+                      "ParameterValue": "${env.PyStache}"
+                  },
+                  {
+                      "ParameterKey": "SecurityGroupIds",
+                      "ParameterValue": "${env.SecurityGroupIds}"
+                  },
+                  {
+                      "ParameterKey": "ServerHostname",
+                      "ParameterValue": "${env.ServerHostname}"
+                  },
+                  {
+                      "ParameterKey": "SubnetIds",
+                      "ParameterValue": "${env.SubnetIds}"
+                  },
+                  {
+                      "ParameterKey": "WatchmakerConfig",
+                      "ParameterValue": "${env.WatchmakerConfig}"
+                  },
+                  {
+                      "ParameterKey": "WatchmakerEnvironment",
+                      "ParameterValue": "${env.WatchmakerEnvironment}"
+                  }
+                ]
+              /
+            }
+        }
+    stage ('Prepare AWS Environment') {
+        options {
+            timeout(time: 1, unit: 'HOURS')
```

Comment:
Potentially redundant given that the `timeout` is already set globally. Probably want to either nuke it from the global `options` set or remove it from the stages' `options` settings.

Reply:
removing from stages